filament/src/materials/skybox.mat

```diff
@@ -35,10 +36,10 @@ fragment {
         if (materialParams.constantColor != 0) {
             sky = materialParams.color;
         } else {
-            sky = vec4(textureLod(materialParams_skybox, variable_eyeDirection.xyz, 0.0).rgb, 1.0);
+            sky = vec4(textureCube(materialParams_skybox, variable_eyeDirection.xyz).rgb, 1.0);
```

Comment:
I can add an `#if`/`#else` here to be safe, or I can leave a comment. Whichever you think is best.

What's the best way to test the performance impact of this change?

Reply:
I would try AGI with a Pixel4 or above, using ANGLE (because AGI only works well with Vulkan). You should be able to see the exact timing of the color pass along with GPU frequency. I don't think you'll see any difference, but I guess better safe than sorry.

Reply:
I ran some tests and found that `texture` was much slower than `textureLod(..., 0.0)`. I opted to add an `#if`/`#else` here.